$NetBSD: patch-curl-config.in,v 1.6 2015/06/30 23:00:21 spz Exp $

- make "curl-config --libs" return proper ldflags for shared libraries

--- curl-config.in.orig	2014-01-05 22:07:54.000000000 +0000
+++ curl-config.in
@@ -148,9 +148,9 @@ while test $# -gt 0; do
            CURLLIBDIR=""
         fi
         if test "X@REQUIRE_LIB_DEPS@" = "Xyes"; then
-          echo ${CURLLIBDIR}-lcurl @LIBCURL_LIBS@
+          echo @LDFLAGS@ ${CURLLIBDIR}-lcurl @LIBCURL_LIBS@
         else
-          echo ${CURLLIBDIR}-lcurl
+          echo @LDFLAGS@ ${CURLLIBDIR}-lcurl
         fi
         ;;
 
